### ISSUE TITLE
PSP34 Transfer event parameter "from" is unnecessary 

### ISSUE THREAD

**Issue by Nradko** [w3f/PSPs#69]
In PSP34  the Transfer event possesses field "from" that is unnecessary.

One can always deduce the previous owner of a token with an ID based on the previous Transfer event that contains the ID.
If such a previous event doesn't exist it means that the token hasn't been minted.

I would recommend changing the standard. Changes to events should not break the cross-contract calls. Also If one assumes the new standard in one's off-chain application the application will be compatible with the previous standard as long as one allows the field "from" to be undefined.

**Comment by deuszx** [comment]
I have to disagree that it's unnecessary. Events are often observed in isolation so information about _past_ events are not readily available. Not all indexers/event listeners have access to the previous `Transfer` event that could have happened months/years ago.
